src/phasorpy/phasor.py

```diff
@@ -2099,6 +2103,91 @@ def phasor_to_principal_plane(
     )
 
 
+def phasor_filter(
```

Comment:
`'phasor_filter'` needs to be added to `__all__`, otherwise the function will not be listed in the documentation.

Then this PR can be merged.